docs/data/sql_functions.yml

```diff
@@ -83,7 +83,7 @@ logical:
     table: BOOLEAN1 && BOOLEAN2
     description: Returns TRUE if BOOLEAN1 and BOOLEAN2 are both TRUE. Supports three-valued logic. E.g., true && Null(BOOLEAN) returns UNKNOWN.
   - sql: NOT boolean
-    table: '!BOOLEAN'
+    table: BOOLEAN.not() or '!BOOLEAN' (Scala only)
```

Comment:
I originally pointed this out as well (just deleted the comment yesterday). The reason I thought this isn't necessary is because `and` and `or` as prefix functions take arbitrarily many arguments (opposed to the postfix ones which are unary). However, `not` is always unary, and a prefix version of it thus doesn't seem as useful. Happy to add it if you still feel otherwise, though.

Reply:
I've added it as a fixup commit to the hotfix. Feel free to either squash it in or discard the commit.